### DISCUSSION TITLE
Drawing a triangle in Uno

### DISCUSSION THREAD

**Discussion by usernamedd** [unoplatform/uno#8414] [Q&A] [answered]
## What would you like to be added:
I want support to draw a custom shape like triangle 
## Why is this needed:
In wpf I can use LineGemetry to make a custom geometry of triangle. If i migrate my wpf project to wasm , I expect custom drawing by using custom shape . 
## For which Platform:

- [x] WebAssembly

## Anything else we need to know?
I find an error message : Windows.UI.Xaml.Media.LineGeometry is not implemented in uno in vs2022.

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the report. `LineGeometry` is implemented in Uno: https://playground.platform.uno/#6f8f7c16

Where did you read that `LineGeometry` is not implemented?

<blockquote></blockquote>

**Reply by usernamedd** [reply]
> Thanks for the report. `LineGeometry` is implemented in Uno: https://playground.platform.uno/#6f8f7c16
> 
> Where did you read that `LineGeometry` is not implemented?

After I restart VS ,the error messages go away . I press CTRL+left mouse button , the resharp shows code like below : 
`[NotImplemented(new string[] {"__ANDROID__", "__IOS__", "NET461", "__SKIA__", "__NETSTD_REFERENCE__", "__MACOS__"})]
    public LineGeometry() => ApiInformation.TryRaiseNotImplemented("Windows.UI.Xaml.Media.LineGeometry", "LineGeometry.LineGeometry()");`
but no error message prompt ,It is wired. 
Thank you again .

**Reply by jeromelaban** [reply]
I see. In this case, it's true that `LineGeometry` is not implemented for the other platforms, but it is implemented for WebAssembly.
